bucardo.schema

```diff
@@ -1969,9 +1969,8 @@ FOR EACH ROW EXECUTE PROCEDURE bucardo.bucardo_delta_names_helper();
         ## Delete any tables that are no longer in the database.
         $dbh->do(q{
             DELETE FROM bucardo.bucardo_delta_targets
-             WHERE NOT EXISTS (
-                SELECT oid FROM pg_class WHERE oid = tablename
-            )});
+             WHERE NOT EXISTS (SELECT oid FROM pg_class WHERE oid = tablename)
```

Comment:
Yeah, the second submission is purely cosmetic. Looking at my submission, it wasn't keeping the local style conventions, so I fixed that.